### ISSUE TITLE
[Bug] Not sure if this work with proton (using latest version + winevfs)

### ISSUE THREAD

**Issue by Nanodragon999** [qsniyg/ksp_stuff#23]
So i tried the latest version of movfs4l.py and winevfs using the --run and --winevfs options thanks to one of [those issue](https://github.com/ajventer/ksp_stuff/pull/19) 's comment.

So far i tried `~/ksp_stuff/movfs4l.py --mo_installroot ~/.steam/debian-installation/steamapps/common/Skyrim/Mod.Organizer.2.2.0 --game Skyrim --winevfs ~/winevfs/bin/winevfs --run '~/.steam/debian-installation/steamapps/common/Proton\ 4.11/proton' run ModOrganizer.exe`

although it did seems to not display much error in the terminal, when i tried using SkyrimLauncher.exe in the MO app, it didn't show any of the enabled mods in the Data entry.(Which it always did on windows) 

I can't think of much ways to really know if movfs4l.py and winevfs is actually working or maybe its on proton side.
Also, i used the aforementioned command in a terminal shortcut from steam, using this `gnome-terminal && echo %command% >/dev/null` as a launch option for SkyrimLE on steam.
It work great for launching most tools and Skyrim itself (without Modorganizer and movfs4l.py/winevfs).

I'm on Pop_OS! 64bit (Ubuntu distro) with latest steam, 2.X Mod Organizer and latest repo from winevfs and movfs4l.py.
(Btw, sorry in advance if i shouldn't have posted this here, since it reference [another project](https://github.com/qsniyg/winevfs) but wasn't sure if i should post this here or over qsniyg's repo.

**Comment by jarrard** [comment]
I gotta give this project another go soon. I do wonder if a launcher gui app can be made to simplify all these commands in a visual way?

**Comment by Nanodragon999** [comment]
probably yeah, there [this](https://github.com/ajventer/ksp_stuff/issues/22) @jarrard 
although my problem here being i'm not sure if this actually work with the aforementioned commands used with proton.
I know i could use wine from the dist/bin/ dir inside the Proton dir, but honestly, i wouldn't want to do that even if it worked for the following reason:
-Prefer to use proton script since it has many diff env i can use, such as setting esync or dx version i want.
So yeah, using proton with just Skyrim and skse work fine, its when using movfs4l and winevfs where i don't know if it actually work, since it doesn't show an error saying it doesn't (and looking back at how the launcher doesn't show the mods used, i'm pretty sure it doesn't work)

**Comment by jarrard** [comment]
you can just load up skyrim xedit to see if the vfs mounted, no need to launch the game.

**Comment by Nanodragon999** [comment]
actually a nice idea, didn't thought of that
But supposing it doesn't work, i guess i'll be clueless as to which command i should try hmm
gonna reboot and try

**Comment by Nanodragon999** [comment]
okay, rebooted
Seems vfs isn't mounted as it should for some reason, even though movfs4l/winevfs didn't show any error for that matter (tried xedit and it didn't show any plugins, while i enabled all of them over MO)

**Comment by jarrard** [comment]
Hmm, maybe some logging output would help. also winevfs might not be persistent during boots.

**Comment by Nanodragon999** [comment]
hmm, so it seems everything work just fine if i use 1.33 Modorganizer instead of the latest version with proton+movfs4l+winevfs
it even show the mods i was using on all front :o so i guess that's solved? unless it should have worked with the latest MO version, but, i guess i'm fine with the older version too...

**Comment by qsniyg** [comment]
@Nanodragon999 Sorry, I didn't document it clearly enough. Don't run ModOrganizer under winevfs, that's an almost surefire way to break your MO profile unfortunately (I should probably add a warning for it).

The way it's meant to be used is to run ModOrganizer separately to manage your mods, then run Skyrim with winevfs (and make sure MO is closed before doing this). It's not very elegant (it still doesn't let you run Skyrim from MO unfortunately), but it's the only way that I really know how to make it work, as wineserver itself needs to be hooked (which will break MO, if it's running).

I'll try to work on some documentation to hopefully clarify this :)

**Comment by Nanodragon999** [comment]
I see :) thanks a lot for making this though, it really complete well with what movfs4l do
Honestly somehow managed to make everything work, need to do more testing but, i think launching Skyrim from MO work fine atm.

**Comment by qsniyg** [comment]
@Nanodragon999 Yeah, MO 1.33 should work without issue :) It's MO2 that updated USVFS that somehow broke Wine, for unknown reasons.
